lib/linux_admin/security.rb

```diff
@@ -0,0 +1,18 @@
+module LinuxAdmin
+  class Security
+    def self.scap_lockdown
+      SshdConfig.apply_scap_settings
+      Service.apply_scap_settings
+      SysctlConf.apply_scap_settings
+      LimitsConf.apply_scap_settings
+      Securetty.remove_vcs
```

Comment:
Yeah, I thought about that, but removing access to the vcs didn't seem really SCAP specific.  I could make the method more generic then create an apply_scap_settings method or just have one that calls remove_vcs with the intention of creating other methods to edit this file in the future.